build/version.go

```diff
@@ -37,7 +37,7 @@ func BuildTypeString() string {
 }
 
 // BuildVersion is the local build version
-const BuildVersion = "1.15.2-dev"
+const BuildVersion = "1.15.2"
```

Comment:
lets add a commit on top, fixing our version?

Reply:
I assume this will be 1.16.0-dev, yes, @jennijuju? I'll punch that in. Otherwise we can adjust.